scmeta/src/scmeta.cr

```diff
@@ -104,12 +100,24 @@ hyperfine = File.open("#{hyperfine_file}") do |file|
   JSON.parse(file).on_steroids!
 end
 
+def pi_accuracy(input : String)
+  # https://www.wolframalpha.com/input?i=N%5BPi%2C+32%5D
+  const_pi = BigFloat.new(3.1415926535897932384626433832795)
```

Comment:
I'm not familiar with Crystal, but if it's like julia then the conversion of a 64-bit floating point literal number to an arbitrary precision number is only going to give you garbage after the ~16th significant digit, at which point you can just stick to plain 64-bit floating point numbers

Reply:
And judging by
![image](https://user-images.githubusercontent.com/765740/198158632-f9950673-3ea1-45dd-b212-141325b6f8fd.png)
I believe I'm right.

Reply:
@giordano fair point. It doesn't even seem to help to [set the precision](https://crystal-lang.org/api/1.6.1/BigFloat.html#new%28num%3AFloat%2Cprecision%3AInt%29-class-method).
![grafik](https://user-images.githubusercontent.com/1914063/198159535-fd618026-171e-4aea-ab05-22fa63b47e06.png)

That's probably why the [constant they have uses 16 digits](https://crystal-lang.org/api/1.6.1/Math.html). Then I'll be using that.

Reply:
If anything, you should use the [string constructor](https://crystal-lang.org/api/1.6.1/BigFloat.html#new%28str%3AString%29-class-method), like you do at the line below for the output of the program.